===================================================================
RCS file: /cvs/gnome/evince/shell/ev-view.c,v
retrieving revision 1.194
retrieving revision 1.195
diff -u -r1.194 -r1.195
--- ev-view.c	2006/03/05 16:08:21	1.194
+++ evince/shell/ev-view.c	2006/03/21 17:54:29	1.195
@@ -574,9 +574,11 @@
 
 	if (view->presentation) {
 		switch (scroll) {
+			case EV_SCROLL_PAGE_BACKWARD:
 			case EV_SCROLL_STEP_BACKWARD:
 				ev_view_previous_page (view);
 				break;
+			case EV_SCROLL_PAGE_FORWARD:
 			case EV_SCROLL_STEP_FORWARD:
 				ev_view_next_page (view);
 				break;
